src/Collection.php

```diff
@@ -1099,30 +987,13 @@ public function updateOne($filter, $update, array $options = [])
      */
     public function watch(array $pipeline = [], array $options = [])
     {
-        if (! isset($options['readPreference']) && ! is_in_transaction($options)) {
-            $options['readPreference'] = $this->readPreference;
-        }
-
-        $server = select_server($this->manager, $options);
-
-        /* Although change streams require a newer version of the server than
-         * read concerns, perform the usual wire version check before inheriting
-         * the collection's read concern. In the event that the server is too
-         * old, this makes it more likely that users will encounter an error
-         * related to change streams being unsupported instead of an
-         * UnsupportedException regarding use of the "readConcern" option from
-         * the Aggregate operation class. */
```

Comment:
I assume this is entirely irrelevant since PHPLIB requires MongoDB 3.6+ and change streams are always supported (assuming we're not a standalone).